minifi/minifi-integration-tests/src/test/resources/c2/protocol/minifi-edge3/expected.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "pattern": "200 OK https://c2-authoritative:10443/c2/config/heartbeat"
+  },
+  {
+    "pattern": "\"operation\":\"UPDATE\",\"operand\":\"CONFIGURATION\""
+  },
+  {
+    "pattern": "200 OK https://c2-authoritative:10443/c2/config\\?class=raspi4"
+  },
+  {
+    "pattern": "200 OK https://c2-authoritative:10443/c2/config/acknowledge"
+  },
+  {
+    "pattern": "^__testTextRaspi4__$"
```

Comment:
Thanks for this, I know it wasn't straightforward but this is really important that now we are validating the parallel handling / publishing of different agent classes.